src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.AbstractIntroduceVariableCodeAction.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.IntroduceVariable;
 
 internal abstract partial class AbstractIntroduceVariableService<TService, TExpressionSyntax, TTypeSyntax, TTypeDeclarationSyntax, TQueryExpressionSyntax, TNameSyntax>
 {
-    private abstract class AbstractIntroduceVariableCodeAction : CodeAction
+    private sealed class IntroduceVariableCodeAction : CodeAction
```

Comment:
pointless abstract type.  subclasses did nothing.

Reply:
nit: rename class

Reply:
you read my mind :)